docs/manual-EN/2.query-language/3.language-structure/user-defined-variables.md

```diff
@@ -0,0 +1,16 @@
+# User-Defined Variables
+
+**Nebula Graph** supports the concept of user-defined variables, which allows passing of a value from one statement to another. A user-defined variable in written as `$ var_name`, `var_name` is the name of variable and can consist of alphanumeric characters, any other characters are not recommended currently.
+
+User-defined variables can only be used in one execution (compound statements separated by semicolon (;) or pipe (|) and be submitted to server to execute together).
+
+Be noted that user-defined variables are session and execution specific. A user-defined variable in one statement can NOT be used in neither other clients nor other executions, which means that the definition statement and the statements that use it should be submitted together. And when the session ends these variables are automatically expired.
```

Comment:
A user-defined variable in one statement can NOT be used in other clients and executions, which means that the variable definition statement and the statements that use it must be submitted together. When the client sends the execution, the variable is automatically released.